.travis.yml

```diff
@@ -18,20 +18,21 @@ cache:
 
 matrix:
   include:
+    - php: 7.3
+      env: WP_TRAVISCI=phpcs WP_VERSION=latest
```

Comment:
Happy to remove some of the other versions but they're a handy check we're not using functionality not allowed in these versions.